main.go

```diff
@@ -9,11 +9,15 @@ package main
 
 import (
 	"fmt"
+	"io"
 	"log"
 	"net/http"
 	"os"
+	"time"
 )
 
+var b io.Writer
```

Comment:
Can we call this `log` instead of `b`, please?

Reply:
done